src/main/kotlin/com/openlattice/hazelcast/pods/HazelcastQueuePod.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2019. OpenLattice, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * You can contact the owner of the copyright at support@openlattice.com
+ *
+ *
+ */
+package com.openlattice.hazelcast.pods
+
+import com.kryptnostic.rhizome.pods.hazelcast.BaseHazelcastInstanceConfigurationPod
+import com.openlattice.hazelcast.HazelcastQueue
+import com.openlattice.ids.HazelcastIdGenerationService
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class HazelcastQueuePod {
+
+    @Bean
+    fun defaultQueueConfigurer(): QueueConfigurer {
+        return QueueConfigurer(BaseHazelcastInstanceConfigurationPod.defaultQueueName) { config ->
+            config.setMaxSize(10000).emptyQueueTtl = 60
```

Comment:
why are we keeping this as the default?

Reply:
would prefer extracting to a constant at least 👍

Reply:
I'm not sure about the values (I kept them as they were), especially why we have a `ttl`, but in case there is no config defined for a queue, I believe this default will apply.

Reply:
After speaking with @geekbeast , the ttl will be deleted